ceph-salt-formula/salt/_modules/ceph_salt.py

```diff
@@ -176,3 +177,26 @@ def __check_unit(unit_name):
     elif out in ['failed', 'auto-restart']:
         state = 'error'
     return (enabled, installed, state)
+
+
+def hostname():
+    try:
+        process = subprocess.run(['hostname'],
```

Comment:
Any reason to NOT use an implementation similar to the one from `cephadm`?

- https://github.com/ceph/ceph/blob/master/src/cephadm/cephadm#L2780-L2782
- https://github.com/ceph/ceph/blob/master/src/cephadm/cephadm#L1214-L1216

Reply:
> Any reason to NOT use an implementation similar to the one from `cephadm`?

They appear to behave the same, so I guess not:

```
master:~ # hostname
master
master:~ # python3
Python 3.6.10 (default, Dec 19 2019, 15:48:40) [GCC] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import socket
>>> socket.gethostname()
'master'
>>> 
master:~ # hostnamectl --set-hostname master.ses7.test
hostnamectl: unrecognized option '--set-hostname'
master:~ # hostnamectl set-hostname master.ses7.test
master:~ # hostname
master.ses7.test
master:~ # python3
Python 3.6.10 (default, Dec 19 2019, 15:48:40) [GCC] on linux
Type "help", "copyright", "credits" or "license" for more information.
>>> import socket
>>> socket.gethostname()
'master.ses7.test'
```